ArenaControl/Stage3.cpp

```diff
@@ -293,7 +296,7 @@ static boolean inCenter(long enc)
    int relativeEncoder;
    
    /* Calculate the relative value +/- from center position */
-   relativeEncoder = (int) (enc % ONE_REVOLUTION);
+   relativeEncoder = (int) (abs(enc) % ONE_REVOLUTION);
```

Comment:
Also a good catch - I thought I had handled this case with the lines below and the final abs(), but this case slips through.

Thanx for the help!